test/unit/range/views/async_input_buffer_test.cpp

```diff
@@ -96,7 +96,7 @@ TEST(async_input_buffer, destruct_with_full_buffer)
         std::this_thread::sleep_for(std::chrono::milliseconds(100));
     } // thread sync at destruction of v1; tests working destruction with full buffer
 
-    EXPECT_GE(std::ranges::distance(v0), 17); // total of at most 10 chars consumed
+    EXPECT_GE(std::ranges::distance(vec), 17); // total of at most 10 chars consumed
```

Comment:
@rrahn Can you give directions how to change this test?

Reply:
I have worked a little more on this, but still can't get an handle on it.
my thoughts:
a) ez! 
b) where should the dummy variable sit? and what should it measure?
c) This doesn't work, since the elements them self don't get moved. There are only references pointed to them passed around.